app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+from os import (
+    makedirs,
+    path
+)
+
+from shutil import move
+
+
+class MoveCommandSyntaxError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+
+    if command[0] != "mv":
+        raise MoveCommandSyntaxError("Move command not found")
+
+    if len(command) != 3:
+        raise MoveCommandSyntaxError(
+            "Invalid move command syntax.\n"
+            "For a successful moving of file "
+            "the command should look like this: "
+            "mv file.txt some_dir/new_file.txt"
+        )
+
+    file_to_move = command[1]
+
+    if not path.exists(file_to_move):
+        raise FileNotFoundError(f"File {file_to_move} does not exist")
+
+    try:
+        with open(file_to_move) as source:
+            source.read()
+    except PermissionError:
+        raise PermissionError(f"No permission to read the file {source}")
```

Comment:
Why here u use message? Like default message from `PermissionError` is  the same, why in except u raise `PermissionError` manually? think about correct re-rasing

Reply:
We have info about correct re-rasing on mate platform